tests/views/test_collection_exercise.py

```diff
@@ -519,6 +519,120 @@ def test_collection_exercise_view_eq_both_ref_dates(self, mock_request):
         self.assertIn("221_201712".encode(), response.data)
         self.assertIn("Set as ready for live".encode(), response.data)
 
+    def load_eq_and_seft_survey(
+        self,
+        mock_request,
+        survey_short_name,
+        collection_exercises,
+        collex_details,
+        collex_events,
+        sample_summary_id,
+        sample_summary,
+        eq_ci,
+        eq_ci_selectors,
+    ):
+        mock_request.get(url_get_survey_by_short_name, json=survey_short_name)
+        mock_request.get(url_ces_by_survey, json=collection_exercises)
+        mock_request.get(url_ce_by_id, json=collex_details)
+        mock_request.get(url_get_collection_exercise_events, json=collex_events)
+        mock_request.get(url_link_sample, json=[sample_summary_id])
+        mock_request.get(url_get_sample_summary, json=sample_summary)
+        mock_request.get(f"{url_get_collection_instrument}?{ci_search_string}", json=eq_ci, complete_qs=True)
+        mock_request.get(
+            f"{url_get_collection_instrument}?{ci_type_search_string_eq}", json=eq_ci_selectors, complete_qs=True
+        )
+
+    @requests_mock.mock()
+    def test_collection_exercise_view_eq_and_seft_non_ref_date(self, mock_request):
```

Comment:
If you rationalise as above you don't need to do all these tests, just 1 for EQ_AND_SEFT, and it can be any (with, without, both )because realistically you are just testing the If in bit